src/NUnitFramework/nunitlite/DebugWriter.cs

```diff
@@ -32,6 +32,7 @@ namespace NUnitLite
     /// output to Debug. We don't use Trace because
     /// writing to it is not supported in CF.
     /// </summary>
+    [Obsolete("No longer used", true)]
```

Comment:
One small request here: can you please either remove the second parameter or explicitly set it to `false` ?

Since this is a public class, people may currently be using it in their tests. Passing `true` here will cause a compiler error for them, potentially "breaking" their build. It's going to be less disruptive if this triggers a warning instead, so that they have an opportunity to move away from using this.

Reply:
Sure thing!

Reply:
@stevenaw removed the parameter.